fast64_internal/oot/ObjectList.xml

```diff
@@ -0,0 +1,382 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Table KeyType="System.UInt16" ValueType="System.String">
+  <Object ID="OBJECT_HUMAN" Key="obj_human" Name="Aria - HUMAN"/>
```

Comment:
I thought you wanted debug names in the actual name 🤔

> the blender name should be constructed from Name and ID without the OBJECT_ prefix

isn't this what I did? I don't understand lol

Reply:
I agree with the result, it's the journey that I think could be better

I mean in blender instead of `get("Name")` = `"Aria - HUMAN"` it could be `get("Name") + " - " + get("ID").replace("OBJECT_", "")` = `"Aria" + " - " + "HUMAN"` = `"Aria - HUMAN"`

Reply:
ahh ok I see, I'll change that then (and I'll do it in the PR too I guess, for consistency)